### PR TITLE
added ten noun plus actions

### PR DESCRIPTION
**Pull request by lornebb**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  Noun:walk
 //  action 1:
-
+walk.forwards
 //  action 2:
-
+walk.backwards
 //  action 3:
- 
+walk.ontheSpot
```
Comment (matkatmusic) [inline main.cpp:133]: These are all missing the thing that makes them a function/action and the thing that indicates the end of the statement.
- [ ] revise.

File: main.cpp [hunk 2]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-
-//  action 3:
- 
+//  Noun: knuckle
+//  action 1: knuckle bends
+knuckle.bend();
+//  action 2: knuckle straightens 
+knuckle.straighten();
+//  action 3: knuckle breaks
+knuckle.break();
```
Comment (matkatmusic) [inline main.cpp:61]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

File: main.cpp [hunk 3]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-//  action 2:
-walk.backwards
-//  action 3:
-walk.ontheSpot
+//  action 1:player walks forwards
+walk.forwards();
+//  action 2:player walks backwards
+walk.backwards();
+//  action 3:player walks on the spot
+walk.ontheSpot();
```
Comment (matkatmusic) [inline main.cpp]: `walk` is an action, not a noun.
`player` is your noun here.
- [ ] revise.

File: main.cpp [hunk 4]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-//  action 2:
-playGuitar.fingerPicked
-//  action 3:
-playGuitar.openChords
+//  action 1:guitar gets played palm muted
+playGuitar.palmMuted();
+//  action 2: guitar gets played finger picked
+playGuitar.fingerPicked();
+//  action 3: guitar gets played with open chords
+playGuitar.openChords();
```
Comment (matkatmusic) [inline main.cpp]: `playGuitar` is not a noun.  it's an action, followed by the thing that is having the action performed on it.
`guitar` is the noun here.

File: main.cpp [hunk 5]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-//  action 2:
-frequency.lowerPitch
-//  action 3:
-frequency.stop
+//  action 1: the frequency pitch will increase
+frequency.higherPitch();
+//  action 2:the frequency pitch will decrease
+frequency.lowerPitch();
+//  action 3:the frequency will stop oscilating
+frequency.stop();
```
Comment (matkatmusic) [inline main.cpp:117]: `frequency.increasePitch();`
`frequency.decreasePitch();`
`frequency.stopOscillating();`

File: main.cpp [hunk 6]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-guitar.picked
-//  action 2:
-guitar.strummed
-//  action 3:
-guitar.amplified
+guitar.picked();
+//  action 2:guitar is picked
+guitar.strummed();
+//  action 3:guitar is strummed
+guitar.amplified();
```
Comment (matkatmusic) [inline main.cpp]: Your pseudo-code does not match your actions.
- [ ] revise

File: main.cpp [hunk 7]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-//  action 2:
-bird.dives
-//  action 3:
-bird.walks
+//  action 1:bird will fly
+bird.flies();
+//  action 2:bird will dive
+bird.dives();
+//  action 3:bird will hops
+bird.hops();
```
Comment (matkatmusic) [inline main.cpp:85]: your pseudo-code does not match your action's tense.  future tense vs. present tense.

File: main.cpp [hunk 8]
```diff
@@ -70,67 +70,67 @@ book.shelved();
...
-//  action 2:
-state.isSolid
-//  action 3:
-state.isLiquid
+//  action 1:state of matter is a gas
+state.isGas();
+//  action 2:state of matter is a solid
+state.isSolid();
+//  action 3:state of matter is a liquid
+state.isLiquid();
```
Comment (matkatmusic) [inline main.cpp]: your noun does not match your actions.
the action noun is `state of matter`
`is` is commonly used to check if some property of an object is true/false/set/etc. it does not make for a good action.
See instruction 2, which further explains why `is` is a bad choice for an action.

File: main.cpp (hunk 3 above)
Comment (lornebb) [reply main.cpp]: ok. Understood. I'll this again before moving forward. x

File: main.cpp (hunk 5 above)
Comment (lornebb) [reply main.cpp:117]: ah! i understand this more now! ok great! changed this too.

File: main.cpp (hunk 6 above)
Comment (lornebb) [reply main.cpp]: edited to match .

File: main.cpp (hunk 4 above)
Comment (lornebb) [reply main.cpp]: changed this whole thing.

File: main.cpp (hunk 7 above)
Comment (lornebb) [reply main.cpp:85]: Changed this toonow.

File: main.cpp (hunk 8 above)
Comment (lornebb) [reply main.cpp]: no booleans. Got it. Changed this too.

**Pull request merged**